test/parsing/errors/stringLiteralEOF-brief.good

```diff
@@ -0,0 +1,2 @@
+stringLiteralEOF.chpl:1: error: end-of-line in a string literal without a preceding backslash
+stringLiteralEOF.chpl:1: error: end-of-file in string literal
```

Comment:
Both print, is that intended?

Reply:
It's the same behavior as before: https://github.com/chapel-lang/chapel/blob/f091e99c07d044b057f6be9d6b47c3c181b25fb3/test/parsing/vass/wrong-escapes-1.good

It would be easy to change, but personally I consider it pretty reasonable that both separate errors come up here.